id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoServiceImpl.java

```diff
@@ -196,6 +196,10 @@ public class IdRepoServiceImpl implements IdRepoService<IdRequestDTO, Uin> {
 	@Autowired
 	private IdentityUpdateTrackerRepo identityUpdateTracker;
 	
+
+	@Value("${mosip.idrepo.credential.request.enable-convention-based-id:false}")
+	private boolean isCredentialrequestv2;
```

Comment:
Rename variable to convery the actual meaning like, enableConventionBasedCredentialId

Reply:
Fixed in https://github.com/mosip/id-repository/pull/850